docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+
+### Introduction
+
+If you work alone or in a small team, you’re likely to have started working on small programs that slowly grow complex and convoluted. Shit happens and you’ll soon find that your perfectly balanced Lasagna evolves into a heaped bowlful of Spaghetti.
+
+Note: I'm running a [Gatsby workshop](https://www.eventbrite.com/e/gatsbyjs-workshop-develop-and-deploy-your-own-blog-tickets-52432622385?aff=GatsbyBlog) in London, February 2019.
+
+The same goes for those who have worked as part of a large company/team. Branding guidelines come in the form of css style-guides or reusable components. Separating the discrete packages from what’s shared becomes a hassle if everything’s stored together. You may even have to deal with several similar projects thrown about in different repos, which can be an arse-ache to keep track of. These are some of the problems monorepos solve.
+
+![Monorepo Diagram](./images/monorepo-diagram.jpeg)
+
+A monorepo is an architectural design pattern that allows us to maintain several packages within a single repo. This reduces the complexity when dealing with packages that share in-house and third-party dependencies.
+
+Think of monorepos like a well-organised supermarket. Each aisle contains a particular kind of food, making it easy to navigate and find the item of food you want. Supermarkets also contain a variety of different foods so you don’t need to go to a series of specific shops to find your gluten-free, dairy-free banoffee pie (which is very much a [thing](https://glutenfreecuppatea.co.uk/2016/02/14/recipe-healthy-banoffee-pie-gluten-free-vegan-refined-sugar-free-dairy-free/)).
+
+With monorepos the code is split into specific packages (aisles). Meaning you can easily navigate the project to find the component or module you want to work on. It also means not needing to maintain separate repos when making changes that affect multiple parts of your stack (the single shop).
+
+Everything is accessible from a single place, while still being organised enough to navigate painlessly.
+Enough with the theory, let’s move on to the tutorial, which is split into 2 sections: development and deployment.
+
+Development — We’ll configure an existing application, built with Gastby, into a monorepo. If you haven’t heard of Gatsby before, then take some time to [read up on it](https://www.gatsbyjs.org/). I won’t delve too much into its implementation details because it’s beyond the scope of this article. We’ll attend the lavish party that the great Gatsby throws for us and we won’t ask why or how they did it. Another tool we’ll use is [Lerna](https://github.com/lerna/lerna), which will manage the dependencies of our project. Lerna will also allow us to link packages together which we’ll use to share components.
```

Comment:
*Development* — We’ll configure an existing application, built with [Gatsby](https://www.gatsbyjs.org/), into a monorepo. 

Another tool we’ll use is [Lerna](https://github.com/lerna/lerna), which will manage the dependencies of our project. Lerna will also allow us to link packages together which we’ll use to share components.

Reply:
I can tighten the paragraph up, but I'd like to still emphasise that this article won't be covering the basics of Gatsby. That's important to see the reader's expection